packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.html

```diff
@@ -0,0 +1,85 @@
+<div class="widget-presentation">
+
+    <h2 class="widget-presentation-title" data-translate="xdsl_access_comfort_exchange_title"></h2>
+
+    <tuc-toast-message></tuc-toast-message>
+
+    <div data-ng-if="$ctrl.isRMAOpened" class="mb-3">
+        <span class="row" data-translate="xdsl_access_comfort_exchange_already_opened_rma"></span>
```

Comment:
You do need `row`class for that `<span>` ? Maybe an helper could be useful (like ml-x)

Reply:
yes, you're right.